src/connection_manager.c

```diff
@@ -156,6 +157,44 @@ static qd_config_sasl_plugin_t *qd_find_sasl_plugin(qd_connection_manager_t *cm,
     return 0;
 }
 
+
+/**
+ * qdcm_normalize_filename
+ *
+ * If the infile (file name) is absolute or null, simply pass it through.  If it is 
+ * relative (contains no slashes or ".."), check to see if the file is in the temporary
+ * store.  If it is, return the full path to the location of the file in the temporary
+ * store.
+ *
+ * If a new string is allocated to handle the return value, free infile.
+ *
+ * Set the qd_error if there is an error condition.
+ */
+static char* qdcm_normalize_filename(char* infile)
```

Comment:
Why is this static function called qdcm_normalize_filename? Why not just qd_normalize_filename. (I am assuming "cm" stands for connection manager? Should we rename other static functions in this file as qdcm_ ?

Reply:
We should probably establish a convention for naming static functions.